### PR TITLE
commit message added intellij ultimate 2nd commit

### PR DESCRIPTION
**Pull request by See29ram**
pull request comment-added ultimate in tools file pls review it

### CODE REVIEW

File: tools [hunk 1]
```diff
@@ -1,2 +1,2 @@
 eclipse
-intellij
+intellij ultimate
```
Comment (tubelightdna) [inline tools:2]: can we change it ultitmate?

Comment (tubelightdna) [inline tools:2]: add ultimate

Comment (tubelightdna) [inline tools:2]: Are guys see my review?

Comment (sea29ram) [inline tools:2]: add reveiw by sea29ram why dont we use vs code

File: tools [hunk 2]
```diff
@@ -1,2 +1,2 @@
 eclipse
```
Comment (tubelightdna) [inline tools:1]: where is eclipse version

**Review by sea29ram — COMMENTED** [review]
added my review comments pls check

**Pull request merged**